Composer/packages/client/src/shell/useShell.ts

```diff
@@ -1,15 +1,31 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { useContext, useMemo, useRef } from 'react';
+import { useMemo, useRef } from 'react';
```

Comment:
@yeze322 could you validate if the shell api operations are working in this PR and you dont see anything wrong with these changes